pkg/gadget-service/api/api.proto

```diff
@@ -50,6 +50,29 @@ message GadgetRunRequest {
   int64 timeout = 13;
 }
 
+message OCIGadgetRunRequest {
+  string url = 1;
+
+  // params is a combined map of all params a gadget could need (including those
+  // of runtime and operators, which need specific prefixes, see implementation in
+  // pkg/runtime/grpc)
+  map<string, string> params = 2;
+
+  // args are all parameters which were not specified with a flag
+  repeated string args = 3;
+
+  // a list of nodes the gadget should run on; if not specified, it should run
+  // on all nodes
+  repeated string nodes = 10;
```

Comment:
In this PR, `nodes` is always set to nil and it is never read. So I suggest to remove it. It can be added in a separate PR if it gets implemented.